cmd/grpcurl/grpcurl.go

```diff
@@ -55,7 +56,11 @@ var (
 		Print version.`))
 	plaintext = flags.Bool("plaintext", false, prettify(`
 		Use plain-text HTTP/2 when connecting to server (no TLS).`))
-	insecure = flags.Bool("insecure", false, prettify(`
+	usealts = flags.Bool("alts", false, prettify(`
+		Use Application Layer Transport Security (ALTS) when connecting to server.`))
+	altsHandshakerServiceAddress = flags.String("alts-handshaker-service", "", prettify(`If set, this server will be used to do the ATLS handshaking.`))
+	altsTargetServiceAccounts    multiString
```

Comment:
Maybe move the new stuff down to line 74.5?

I also wouldn't be opposed to adding a few line breaks to visually separate clusters of related options

Reply:
Done